lib/pbench/server/auth/auth.py

```diff
@@ -166,13 +242,13 @@ def verify_third_party_token(auth_token: str, oidc_client: OpenIDClient) -> bool
 
         if not oidc_client.TOKENINFO_ENDPOINT:
```

Comment:
Is it really reasonable for an OIDC provider not to support online verification? That sounds weird ... is this really a "warning in user authentication" or an "error in OIDC configuration"??

Reply:
If the client is `public` then the Keycloak doesn't allow the online token verification as this requires network invocation to the Keycloak server and could possibly overload the server. It's easy for `public` client users to abuse this endpoint and so online validation is reserved only for the `non-public` clients to potentially avoid this issue.

Reply:
Fascinating. A comment might be useful to clarify that.